.gitmodules

```diff
@@ -27,4 +27,8 @@
 [submodule "content-modules/opentelemetry-go"]
 	path = content-modules/opentelemetry-go
 	url = https://github.com/open-telemetry/opentelemetry-go
-  go-pin = v1.23.0-rc.1-31-geabcef4c2
+    go-pin = v1.23.0-rc.1-31-geabcef4c2
+[submodule "content-modules/opentelemetry-java-examples"]
+	path = content-modules/opentelemetry-java-examples
+	url = git@github.com:zeitlinger/opentelemetry-java-examples.git
```

Comment:
this should be replaced once https://github.com/open-telemetry/opentelemetry-java-examples/pull/422 is merged